### DISCUSSION TITLE
Error in spades result

### DISCUSSION THREAD

**Discussion by Noormaho** [ablab/spades#980] [Q&A]
I denovo assembled the Illumina NGS sequence reads via SPAdes. The SPAdes gave contigs assembled files more than 12 MB, however actual genome size of S. Typhi is 4 MB. What could be the reason of generating so large size denovo contigs assembly file? 
Kindly acknowledge. 

Kind regards, 
Ms Mah Noor Mumtaz

**Comment by asl** [comment]
I would suggest you to check for possible contamination

**Reply by Noormaho** [reply]
Thanks for your reply. When I submitted these assembled data to ncbi genome so they show contaminated contigs so I removed all those contigs but after removing contaminated contigs or nodes again they show genome size error. Will you please guide me how to resolve this issue.

Kind regards 
Ms mahnoor mumtaz

**Reply by asl** [reply]
Well, you need to study what is inside your data. SPAdes simply assembles everything you're having

**Comment by Noormaho** [comment]
Dear,
Thanks for your reply. When I submitted these assembled data to ncbi genome
so they show contaminated contigs so I removed all those contigs but after
removing contaminated contigs or nodes they show genome size error that
your assembled sequence size is 12Mb while S.typhi genomesize is 4.7Mb.
Will you please guide me how to resolve this issue.

Kind regards
Ms mahnoor mumtaz

**Reply by asl** [reply]
No need to repeat you answer. And "ncbi genome" is not the silver bullet. I would suggest to find someone who will help you analyzing your data. Assembler is not a magic tool, it will produce whatever is contained in the input. If the expected genome size does not match, then you'd need to analyze your data and look for possible sources / reasons of contamination.
